### ISSUE TITLE
New recipe: Hot sauce

### ISSUE THREAD

**Issue by jeremyckahn** [jeremyckahn/farmhand#117]
Once #104 is done, let’s add hot sauce to spice up our recipes!

**Comment by lstebner** [comment]
love it! can we repurpose this issue to be a handful of new jalapeño recipes? hot sauce, which i think can be made from just the peppers. then i'm thinking we add salsa which could be made with pepper, corn, tomato, and onion, and then spicy cheese! cheese and jalapeños. what do you think @jeremyckahn ?

**Comment by jeremyckahn** [comment]
I like all of these ideas! It sounds like you have some specific ideas in mind. If you'd like to spin up a new issue with what you've got in mind, we can close this one as a dupe. 🙂

**Comment by lstebner** [comment]
alright i have added #120 @jeremyckahn

**Comment by jeremyckahn** [comment]
> alright i have added #120 @jeremyckahn

Sweet! Closing this now. 🌶
